components/split-view/__examples__/customItemList.jsx

```diff
@@ -0,0 +1,249 @@
+import React from 'react';
+import IconSettings from '~/components/icon-settings';
+import SplitView from '~/components/split-view/index';
+import SplitViewHeader from '~/components/split-view/splitViewHeader';
+import SplitViewList, { SORT_OPTIONS } from '~/components/split-view/splitViewList';
+import Icon from '~/components/icon';
+import Button from '~/components/button';
+import Dropdown from '~/components/menu-dropdown';
+import DropdownTrigger from '~/components/menu-dropdown/button-trigger';
+import shortid from 'shortid';
+
+const listOptions = [
```

Comment:
As this is for a custom list item you can specify any data structure to correspond to your template